fastapi/datastructures.py

```diff
@@ -1,5 +1,11 @@
 from typing import Any, Callable, Iterable, Type
 
+from starlette.datastructures import URL  # noqa: F401
```

Comment:
I don't know how to create a multiline suggestion, but:
```suggestion
from starlette.datastructures import (
    Address,
    FormData,
    Headers,
    QueryParams,
    State,
    UploadFile as StarletteUploadFile,
    URL
)
```

Reply:
In general I would agree, but the change you suggest is incompatible with the `bash scripts/format-imports.sh` script.

That script invokes `isort fastapi tests docs_src scripts --force-single-line-imports`, which splits each import to a single line, but does not propagate the `noqa` hint to each line. It then invokes `autoflake --remove-all-unused-imports ...` which deletes the imports.